src/mongo/delegates/Tables.class.php

```diff
@@ -362,8 +371,9 @@ public function deleteTableRowsByTableId($tableId) {
             return;
         }
 
+        $query = array("_id.type"=>$tableId);
         $this->config->getCollectionForTable($this->storeName, $tableId)
-            ->remove(array("_id.type"=>$tableId), array('fsync'=>true));
+            ->deleteMany($query);
```

Comment:
do you need to inspect the deleteResult object to verify that it actually worked or did what you expected it to?